backend/src/test/scala/cromwell/backend/wdl/ReadLikeFunctionsSpec.scala

```diff
@@ -35,32 +35,40 @@ class ReadLikeFunctionsSpec extends FlatSpec with Matchers {
 
   it should "correctly report the size of a supplied, optional, 2048 byte file" in {
     val readLike = new TestReadLikeFunctions(Success(2048d))
-    readLike.size(Seq(Success(WomOptionalValue(WomFileType, Some(WomSingleFile("blah")))))) should be(Success(WomFloat(2048d)))
+    readLike.size(Seq(Success(WomOptionalValue(WomSingleFileType, Option(WomSingleFile("blah")))))) should
+      be(Success(WomFloat(2048d)))
   }
 
   it should "correctly report the size of a supplied, optional optional, 2048 byte file" taggedAs PostWomTest ignore {
```

Comment:
TOL so many `ignore`d `PostWomTest`s

Reply:
This one got a ticket https://github.com/broadinstitute/cromwell/issues/2611